app/javascript/stylesheets/application.scss

```diff
@@ -0,0 +1 @@
+@import "bootstrap"
```

Comment:
I wonder why this file is created for since `app/javascript/packs/application.js` already has `import bootstrap`

Reply:
one is js and one is css